community/neo4j/src/test/java/org/neo4j/store/watch/FileWatchIT.java

```diff
@@ -63,7 +63,7 @@
 
 public class FileWatchIT
 {
-    private static final long TEST_TIMEOUT = 30_000;
+    private static final long TEST_TIMEOUT = 600_000;
```

Comment:
That is fully depends from how fast we get notification from underlying file system, that is realistic timeout for aws machines